### DISCUSSION TITLE
Mise à jour des sources dans vStream

### DISCUSSION THREAD

**Discussion by coupagabana** [Kodi-vStream/venom-xbmc-addons#4270] [Support & aide] [answered]
Bonjour tout le monde,

J’ai remarqué que, depuis un certain temps, certaines sources de Vstream ne fonctionnent plus.

Serait-il possible de mettre à jour les sources ?
Ou bien d’ajouter un formulaire dans l’application qui nous permettrait de les actualiser nous-mêmes en ajoutant des sources fonctionnelles ?

Merci d’avance.

**Comment by detobel36** [comment]
Salut,

Le code source est disponible. Si c'est juste un changement d'URL, il suffit de mettre à jour un fichier. Tu peux donc déjà mettre à jour toi dès maintenant si tu connais la nouvelle URL d'une source.
Si tu veux en rajouter une, un formulaire ne suffit pas. Il faut développer.
Soit tu développes toi la solution (tu peux déjà le faire maintenant). Soit tu veux juste proposer une source et tu fais une issue pour ça...

**Reply by coupagabana** [reply]
Bonjour, Merci pour ta réponse

Je ne sais pas comment ajouter les sources. Je me permets donc de proposer que quelqu'un puisse les ajouter. Merci par avance pour votre aide.

https://fusov.com/y3qu3ytbj/home/fusov
https://titrov.com/13tt4k6cj/home/titrov
https://www.iramiv.com/b43ly2wh5o63n/home/iramiv/
https://vvw.french-stream.bio/

https://wiflix-hd.vip/
https://fr.dnfs.lol/
https://fanstream.us/
https://seriescultes.site/
https://pcmirror.cc/home

Animés
https://v5.voiranime.com/
https://animesite.fr/
https://streaming-integrale.com/
https://gum-gum-streaming.com/
https://anime-sama.fr/

Replays
https://www.replay.fr/
https://www.myretrotvs.com/
https://mytaratata.com/

**Reply by sizanic** [reply, marked as answer]
Bonjour,
> https://fusov.com/y3qu3ytbj/home/fusov
https://titrov.com/13tt4k6cj/home/titrov
https://www.iramiv.com/b43ly2wh5o63n/home/iramiv/

Ces 3 adresses sont pour le même site. Ce site est connu pour avoir  100 noms. Ce site est déjà présent dans vStream sous le nom Kepliz.

> https://vvw.french-stream.bio/

Ce site est protégé par CloudFlare, il ne sera pas ajouté.
Pour autant. Un autre site "french-stream" sera bientôt ajouté.

> https://fr.dnfs.lol/

justement, c'est une copie du site qui sera bientôt ajouté.


> https://wiflix-hd.vip/

C'est exactement le site wiflix déjà proposé dans vStream 🤔 
https://github.com/Kodi-vStream/venom-xbmc-addons/blob/961979aa55182dd6d0884933f6a9c4474af7756e/plugin.video.vstream/resources/sites.json#L312

> https://v5.voiranime.com/

C'est exactement le site voiranime déjà proposé dans vStream 🤔 

> https://gum-gum-streaming.com/

C'est exactement le site gum-gum-streaming déjà proposé dans vStream 🤔 
Des liens ne sont pas accessibles, une correction sera bientôt disponible.


Pour les autres propositions, merci. On va regarder ce qu'ils valent.

**Reply by coupagabana** [reply]
Bonjour 

Merci pour votre réactivité, et pour vos efforts louables afin de garder cet addon au top

Ma demande résulte du fait que lorsque je fais une recherche sur un film ou une série un nombre limité de site est inspecté ( 7 pour les série et 10 (je pense) pour les films) alors qu'il y a plusieurs site comme ressources. (Kepliz => OK pour les films, pour les séries il n'est pas dispo au niveau des sites de recherche).

Merci

**Reply by sizanic** [reply]
Est-ce une question concernant Kepliz qui n'est pas dispo au niveau des sites de recherches de séries ?
Tu connais pourtant ce site, tu as donné 3 sites qui en sont une copie. Rends-toi sur ces sites et tu verras qu'il n'a jamais proposé de séries 🤔 

Si tu veux connaitre les sites utilisés pour les recherches de séries : 
vStream/VOD/Séries/sources

pareil pour les films : 
vStream/VOD/Films/sources

C'est le nombre que tu donnes, 7 et 10.

Une nouvelle version de vStream arrive bientôt avec de nouveaux sites 👍🏻
